src/cdk/scrolling/virtual-scroll-fixed-size.ts

```diff
@@ -0,0 +1,83 @@
+import {Range} from '@angular/cdk/collections';
+import {Directive, forwardRef, Input, OnChanges} from '@angular/core';
+import {VIRTUAL_SCROLL_STRATEGY, VirtualScrollStrategy} from './virtual-scroll-strategy';
+import {CdkVirtualScrollViewport} from './virtual-scroll-viewport';
+
+
+/** Virtual scrolling strategy for lists with items of known fixed size. */
+export class VirtualScrollFixedSizeStrategy implements VirtualScrollStrategy {
+  private _viewport: CdkVirtualScrollViewport;
+
+  constructor(public itemSize: number, public bufferSize: number) {}
+
+  /** Initialize the strategy and specify the viewport it will be working with. */
+  init(viewport: CdkVirtualScrollViewport) {
+    this._viewport = viewport;
+    this._viewport.totalContentSize = this._viewport.dataLength * this.itemSize;
+    this._updateRenderedRange();
+  }
+
+  /** Re-initialize the strategy with the same viewport. */
+  reinit() {
```

Comment:
Why is this method necessary?

Reply:
I gave it a better name so its more obvious why its necessary